src/converters/protobufToJsonFactory.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Converts protobuf message to a JSON (factory)
+ *
+ * @param MessageClass
+ * @param GRPCMessageClass
+ *
+ * @returns {Object}
+ */
+function protobufToJsonFactory(MessageClass, GRPCMessageClass) {
```

Comment:
We need consistency. Somewhere we call messages `pb` and `pbjs`. Here we call `MessageClass` and `GRPCMessageClass`. For me personally, they are both `js` and `GRPC`.